### ISSUE TITLE
Use absolute path to `magic.mgc` by default

### ISSUE THREAD

**Issue by 0x2b3bfa0** [npcz/magic#10]
Could this packagage use [`require.resolve`](https://nodejs.org/api/modules.html#requireresolverequest-options) to get the absolute path to the `magic.mgc` file?

```javascript
FileMagic.magicFile = require.resolve("@npcz/magic/dist/magic.mgc");
```

It looks nicer than the current approach, but there might be some drawback I'm not aware of.

**Comment by 0x2b3bfa0** [comment]
By the way: awesome project! ❤️

**Comment by abdes** [comment]
That's a good suggestion, which I will adopt, although it has a small implication. The unit tests and the examples were referring to the local files so we don't have to install the module during development.

It's not a big deal though, as yarn makes it quite simple to use the local files during development but act as if the module was installed.

```shell
$ yarn link
$ yarn link "@npcz/magic"
```

and voila.

**Comment by abdes** [comment]
This is now fully solved in release 1.3.12.

**Comment by 0x2b3bfa0** [comment]
Thank you very much!
